man/systemd.network.xml

```diff
@@ -1561,7 +1561,8 @@
           <term><varname>SendRelease=</varname></term>
           <listitem>
             <para>When true, the DHCPv4 client sends a DHCP release packet when it stops.
-            Defaults to false.</para>
+            Defaults to false. Note that set <varname>KeepConfiguration=</varname>to <literal>no</literal>
```

Comment:
Grammar:

> Note that KeepConfiguration must be set to no in order for this to work as expected.

Reply:
I am not sure which one please :) thanks !